### ISSUE TITLE
list

### ISSUE THREAD

**Issue by nafisa-valieva** [BetaMasaheft/Manuscripts#78]
Ciao Pietro,
now validating, I see that normal encoding of the text (<lb/> <cb/> <pb/> etc )is not allowed in the <list> element. Is there a way to allow it or what shall I do, encoding those documents or notes which are additions according to Andrist and us? I do not want to loose this data. I have started to delete it, but now on the half way through I regret to delete it, I'd better keep it! and I can have it from my file back, if the schema allows.

**Comment by PietroLiuzzo** [comment]
of which `<list>`are you talking about? if you have made a commit, I can check it on your branch.

**Comment by PietroLiuzzo** [comment]
I see now, is additions.
you should have something like this
https://betamasaheft.eu/Guidelines/?q=additions&id=additionsVaria

is it in` <q>` that the elements are not allowed?

**Comment by nafisa-valieva** [comment]
I have made a commit now,
yes, in <q>

**Comment by nafisa-valieva** [comment]
yes, I am following this, but it says that the text is not allowed

**Comment by PietroLiuzzo** [comment]
you are using `<lb/>` as if it was `<l>`. if they are just lines, `<lb/>`  should be empty, and may carry a` @n`.  if they are verses, use `<l>verse line</l>`. Also `<cb/>` and `<pb/>` are fine, but you have `</cb>` and `</pb>` which are unopened closing tags. you `<q>` element is also not closed, but there are so many errors before that it does not even get to that point. Additionally, the `<body>` is missing.
There used to be a transcription here, you do not want to have it any more?

**Comment by PietroLiuzzo** [comment]
By all means, do not delete all this nice encoding! it is just how it should be done really, a lot of god work, and will be even better when the encoding will also be right. :) keep the good work up.

**Comment by nafisa-valieva** [comment]
Thank you, Pietro! I want it in a different way, so I am changing it now.
